src/storage/v2/disk/storage.cpp

```diff
@@ -1487,6 +1487,7 @@ DiskStorage::DiskAccessor::CheckVertexConstraintsBeforeCommit(
       }
 
       const auto &edge = edge_acc.find(gid);
+      // TODO(gitbuda): Doesn't work on the latest compiler, what's the best way to resolve?
```

Comment:
Let's figure what's the proper way to resolve this CC @as51340

Reply:
edge_acc is just accessor to skip list. Nothing specific to disk, so no idea. Try to comment this line and see whether the same thing will fail somewhere else because it should, we have plenty of `find` commands throughout the codebase.